features/languages.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Languages
+@QA @Languages
+Scenario: Ensure you can change languages in CommCare
+	Then I install the ccz app at "languages.ccz"
+	Then I login with username "clark" and password "123"
+
+	#Check that default language deployed
+	Then I press start
+	Then I select module "Basic Form Test"
+	And I select form "Languages"
+
+	Then I see the text "Enter a name:"
+	Then I enter text "English"
+	Then Submit
+	Then I go back to the home screen
+
+	#Change the language from home screen menu
+	Then I select "Change Language" menu item
+	Then I see the text "Hindi"

```

Comment:
Can you also add `Then I see the text "English"` to assert that english is there too?